content/chainguard/chainguard-images/about/development-containers.md

```diff
@@ -0,0 +1,62 @@
+
+These variants are designed for development tasks such as building, testing, or debugging. They can be used to build software artifacts that are then copied into standard images as part of a multi-stage build, or to test workflows interactively in an environment similar to a standard image. Development images contain familiar utilities such as package managers and shells. While our standard images have advantages related to security, development images are also secure and production-ready. Development images are tagged `:latest-dev`.
+
+While we encourage you to use Chainguard's standard container images in your live deployments, development images are useful for many parts of the development lifecycle. This article explains some of the key features of development images and how they differ from our standard  between these variants and outlines ways these variants come together in creating a secure deployment.
+
+## Chainguard Container Security
+
+Chainguard's standard container images have the following advantages:
+
+- Our standard container images contain fewer packages. While Chainguard moves quickly to patch CVEs in all images, our standard, distroless images still experience fewer CVEs overall. Reducing the number of packages also reduces the potential number of unknown vulnerabilities that might apply to an image.
+- Not all executables are created equal. Shells such as bash, package managers such as apk, and communication-ready utilities such as Git and curl are general-purpose tools that are broadly exploitable.
+- A smaller image can use fewer resources and reduce deployment time. In some cases, especially with already-large images, a smaller version can make a deployment more stable or robust.
+- Removing unnecessary components increases the observability and transparency of the image. Reducing the number of components can facilitate risk assessment or post-incident reporting.
+
+While our standard images can be considered to have advantages for security, the development variants of Chainguard Images are also low-to-no CVE, include useful attestations such as SLSA provenance and SBOMs, and follow other security best practices. You should feel comfortable using these secure development images in production if they better fit your use case.
+
+## Using Development Images
+
+Though we encourage the use of Chainguard's standard, non-development container images in your final deployment, development images have many use cases. These include:
+
+- **Building**: In many Dockerfile builds, you will need to generate software artifacts such as static binaries or virtual environments as part of the build process. Development images are ideal for this use case, and after generation artifacts can be copied to a standard image for use. See [How to Port a Sample Application to Chainguard Images](/chainguard/migration/porting-apps-to-chainguard/) for a detailed example.
```

Comment:
"and after generation artifacts" reads awkwardly to me

Reply:
Tried to smooth it out